packages/react/src/components/CardEditor/CardEditForm/CommonCardEditFormFields.jsx

```diff
@@ -226,7 +239,7 @@ const CardEditFormContent = ({
   );
 };
 
-CardEditFormContent.propTypes = propTypes;
-CardEditFormContent.defaultProps = defaultProps;
+CommonCardEditFormFields.propTypes = propTypes;
+CommonCardEditFormFields.defaultProps = defaultProps;
 
-export default CardEditFormContent;
+export default CommonCardEditFormFields;
```

Comment:
This component was using the same name as another component. This caused PropType errors in the newer versions, but it wasn't actually breaking anything because the default export was using the correct name when it was being imported. Just renamed this component to match the filename (and component name where it was actually used).